mlem/cli/apply.py

```diff
@@ -83,7 +86,11 @@ def apply(
     with set_echo(None if json else ...):
         if import_:
             dataset = import_object(
-                data, repo=data_repo, rev=data_rev, type_=import_type
+                data,
+                repo=data_repo,
+                rev=data_rev,
+                type_=import_type,
+                batch=batch,
```

Comment:
for now let's ignore this branch and focus on the other one without import. once we done, we can discuss how to approach this (I am not sure myself)